packages/expo-clipboard/src/Clipboard.ts

```diff
@@ -170,17 +175,36 @@ export async function hasImageAsync(): Promise<boolean> {
  * is a no-op on Web.
  *
  * @param listener Callback to execute when listener is triggered. The callback is provided a
- * single argument that is an object with a `content` key.
+ * single argument that is an object containing information about clipboard contents.
  *
  * @example
  * ```typescript
- * addClipboardListener(({ content }: ClipboardEvent) => {
- *   alert('Copy pasta! Here's the string that was copied: ' + content);
+ * Clipboard.addClipboardListener(({ contentTypes }: ClipboardEvent) => {
+ *   if (contentTypes.includes(Clipboard.ContentType.PLAIN_TEXT)) {
+ *     Clipboard.getStringAsync().then(content => {
+ *       alert('Copy pasta! Here's the string that was copied: ' + content)
```

Comment:
It needs escaping 😄 Template string would be good here too 🤔 
```suggestion
 *       alert('Copy pasta! Here\'s the string that was copied: ' + content)
```